src/assets/translations/fr/main.json

```diff
@@ -1126,7 +1128,9 @@
         "insufficientBalance": "Votre porte-monnaie ne dispose actuellement pas d'un solde en %{symbol} suffisant sur %{chainName} pour exécuter cet ordre. L'ordre est toujours ouvert et deviendra exécutable lorsque vous approvisionnerez votre solde en %{symbol} sur %{chainName}.",
         "insufficientAllowance": "Cet ordre nécessite qu'une allocation supplémentaire en %{symbol} sur %{chainName} soit approuvée pour que CoW Swap puisse exécuter cet ordre. L'ordre est toujours ouvert et deviendra exécutable une fois l'approbation d'allocation terminée."
       }
-    }
+    },
+    "limitOrderFilled": "Ordre à cours limité exécuté",
```

Comment:
Does it really mean something?

Reply:
It was purely informational, I never heard about `Ordre à cours` but it seems like it exists

Reply:
No worries, seems the most used translation, but if everyone would prefer the shorter "Ordre limite" or even "Ordre limité" I could do a pass and change all the occurrences. All three seem to have some kind of usage at times/depending the context. The one I've chosen is the most *formal* I'd say.